src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+
+    public String getRandomBall() {
+        Random randomNumber = new Random();
+        int i = randomNumber.nextInt(100);
```

Comment:
let's create a constant for 100 https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names


```suggestion
        int number = randomNumber.nextInt(MAX_NUMBER);
```